core/src/main/scala/core/database/package.scala

```diff
@@ -0,0 +1,34 @@
+package core
```

Comment:
Package name is prone to name clash not sufficiently distinctive, would rather use orga+name spécifiés un build there

Reply:
I'm looking for a name for the project. But I can't stick myself to one for now so I call it `QueryMonad` in the package name